code/obj/item/instruments.dm

```diff
@@ -105,10 +101,57 @@
 	proc/post_play_effect(mob/user as mob)
 		return
 
+
+	ui_interact(mob/user, datum/tgui/ui)
+		ui = tgui_process.try_update_ui(user, src, ui)
+		if(!ui && use_new_interface == 1)
+			ui = new(user, src, "MusicInstrument")
+			ui.open()
+
+	ui_data(mob/user)
+		..()
```

Comment:
No harm, but would write similarly to `ui_act` if you're going to add a check for `..()`, i.e.
```suggestion
		. = ..()
		if (.)
			return
```

Reply:
This breaks the thing because parent returns `list()` which is truthy.